timesketch/lib/utils_test.py

```diff
@@ -156,23 +176,54 @@ def test_invalid_CSV_file(self):
             # Call next to work around lazy generators.
             next(_validate_csv_fields(mandatory_fields, df_02))
 
+    def test_datetime_parsing_csv_file(self):
```

Comment:
This is the only real new thing in the unit tests, rest is only black formatting